coalib/bearlib/languages/definitions/Markdown.py

```diff
@@ -0,0 +1,10 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class Markdown:
+    extensions = [
+        '.markdown', '.mdown', '.mkdn',
```

Comment:
not `.text` - that will cause too many files to be linted using markdown, which will cause all sorts of errors on normal text files.

Also I wouldnt want to assume that `.Rmd`'s syntax for the code blocks is legal markdown (based on which spec of markdown ... ? ;-)  IMO, exclude it, and someone else can worry about that problem.